cobbler/remote.py

```diff
@@ -3163,6 +3211,20 @@ def __make_token(self, user: str) -> str:
         self.token_cache[b64] = (time.time(), user)
         return b64
 
+    @staticmethod
+    def __is_token(token: str) -> bool:
+        """
+        Simple check to validate if it is a token.
+
+        __make_token() uses 25 as the length of bytes that means we need to padding bytes to have a 34 character str.
+        Because base64 specifies that the number of padding bytes are shown via equal characters, we have a 46 character
```

Comment:
```suggestion
        Because base64 specifies that the number of padding bytes are shown via equal characters, we have a 36 character
```
34 chars (decoded 25 bytes) + `==` = 36